open-science-projects.md

```diff
@@ -23,34 +23,58 @@ Discussions on errors, new features, and other specific issues for riboviz, take
 
-Completing an open science project was a very exciting experience, though rather intimidating as it felt like we were contributing to a larger project rather than completing an isolated experiment or literature review. The idea that everything we did could be seen by others and had the potential to help them fuelled this feeling. Knowing that experts within the field could examine the code we had written or critique the conclusions we had reached created an additional layer of motivation in terms of maintaining high quality work that could be understood by others. Every aspect of the project being freely available also removed the anxiety that would have arisen if we had been dealing with confidential ideas or processes, for example testing potential antibiotics. Instead of hiding trade secrets, we were able to freely collaborate and learn from the work of others. Additionally, there was more room for dialogue with other students completing open science projects. This was a valuable aid as it allowed us to bounce ideas off each other compared to the stress and isolation of a closed project that could not be discussed with peers.
+We had the freedom to join a supportive lab where we were encouraged to take our projects in any direction of interest, while still being guided towards contributing useful material to the wider community.
+The open science nature of the project allowed for wider collaboration with other members of the lab and cooperating with research groups across the world.
+It provided potential for learning a variety of skills as we were free to aid in different sub-projects, in addition to our own, without concerns about confidentiality.
+
+Completing an open science project was a very exciting experience, though rather intimidating as we were contributing to a larger project rather than completing an isolated experiment or literature review.
+The idea that everything we did could be seen by others, and had the potential to help or hinder them, fuelled this feeling of intimidation.
+Knowing that experts within the field could examine the code we had written or critique the conclusions we had reached created an additional layer of motivation in terms of maintaining high quality work that could be understood by others.
+Our project, being open, continued to avoid generating anxiety about dealing with confidential ideas or processes, contrasting with the stress and isolation described by some of our peers working on closed projects. EDWARD ASKS: IS THIS ACCURATE? EDIT OR REMOVE.
```

Comment:
REMINDER: this PR should not be merged before this EDWARD ASKS is resolved & the tag removed.

Reply:
I'm creating a comment for this because it's the kind of thing I would be very likely to forget and the comment makes it easier to see :D

Reply:
Personally I can't think of an example of extra stress or isolation due to confidentiality being described by someone I know. It is more I know if I had been doing a closed experiment then I would not have been able to excitedly talk to friends and family about my project (which was the only thing in my life at that point) because I would think "Oh no! What if I am breaking a rule or causing a problem AHHHH!" anytime I was asked about it. However that may just be me and how my brain works.
I cannot speak for other students so it may be best to remove this sentence.

Reply:
Thanks, that’s clear. I’ve edited out that part.